notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb

```diff
@@ -0,0 +1,883 @@
+{
```

Comment:
<p>I would remove "You make a batch prediction by submitting a batch prediction job.".  it is redundant.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1592/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>